src/CocktailDetails/CocktailDetails.test.tsx

```diff
@@ -1,70 +1,181 @@
-			</MemoryRouter>
-		);
+  it("Should display the fetched cocktail instructions", async () => {
+    const mockToggle = jest.fn();
+    const { getByText } = render(
+      <MemoryRouter>
+        <CocktailDetails
+          id={"11007"}
+          favCocktails={[]}
+          setFavCocktails={Function}
+          toggleUserInteraction={mockToggle}
+          madeCocktails={[]}
+          setMadeCocktails={Function}
+        />
+      </MemoryRouter>
+    );
 
-		const detailInstructions = await waitFor(() => getByText('Fill glass with ice and fish, add vodka, grape soda and orange juice. DO NOT STIR!!!!! Serve well chilled.'));
+    const detailInstructions = await waitFor(() =>
+      getByText(
+        "Rub the rim of the glass with the lime slice to make the salt stick to it. Take care to moisten only the outer rim and sprinkle the salt on it. The salt should present to the lips of the imbiber and never mix into the cocktail. Shake the other ingredients with ice, then carefully pour into the glass."
+      )
+    );
 
-		expect(detailInstructions).toBeInTheDocument();
-	});
+    expect(detailInstructions).toBeInTheDocument();
+  });
 
-	it('Should display the fetched cocktail ingredients matched with their measure', async () => {
-		const { getByText } = render(
-			<MemoryRouter>
-				<CocktailDetails id={'14622'} />
-			</MemoryRouter>
-		);
+  it("Should display the fetched cocktail ingredients matched with their measure", async () => {
+    const mockToggle = jest.fn();
+    const { getByText, getAllByText } = render(
+      <MemoryRouter>
+        <CocktailDetails
+          id={"14622"}
+          favCocktails={[]}
+          setFavCocktails={Function}
+          toggleUserInteraction={mockToggle}
+          madeCocktails={[]}
+          setMadeCocktails={Function}
+        />
+      </MemoryRouter>
+    );
 
-		const ingredient1 = await waitFor(() => getByText('Vodka 1/3 part'));
-		const ingredient2 = await waitFor(() => getByText('Grape soda 1/3 part'));
-		const ingredient3 = await waitFor(() => getByText('Orange juice 1/3 part'));
-		const ingredient4 = await waitFor(() => getByText('Ice lots'));
-		const ingredient5 = await waitFor(() => getByText('Candy 1 dash'));
+    const ingredient1 = await waitFor(() => getByText("Vodka"));
+    const strMeasure1 = await waitFor(() => getAllByText("1/3 part"));
+    const ingredient2 = await waitFor(() => getByText("Grape soda"));
+    const ingredient3 = await waitFor(() => getByText("Orange juice"));
+    const ingredient4 = await waitFor(() => getByText("Ice"));
+    const strMeasure4 = await waitFor(() => getByText("lots"));
+    const ingredient5 = await waitFor(() => getByText("Candy"));
+    const strMeasure5 = await waitFor(() => getByText("1 dash"));
 
-		expect(ingredient1).toBeInTheDocument();
-		expect(ingredient2).toBeInTheDocument();
-		expect(ingredient3).toBeInTheDocument();
+    expect(ingredient1).toBeInTheDocument();
+    expect(strMeasure1.length).toEqual(3);
```

Comment:
Thank you Alex for helping me figure out the length testing, cool stuff I'll tuck away for future testing